pkgs/development/libraries/speechd/default.nix

```diff
@@ -1,38 +1,88 @@
-{ fetchurl, lib, stdenv, intltool, libtool, pkgconfig, glib, dotconf, libsndfile
-, libao, python3Packages
-, withEspeak ? false, espeak
+{ stdenv, lib, pkgconfig, fetchurl, wrapPython
+, intltool, itstool, libtool, texinfo, autoreconfHook
+, glib, dotconf, libsndfile
+, pyxdg
+, withLibao ? true, libao
+, withPulse ? false, libpulseaudio
+, withAlsa ? false, alsaLib
+, withOss ? false
+, withFlite ? true, flite
+, withFestival ? false # TODO: , festival-freebsoft-utils
+, withEspeak-ng ? true, espeak-ng, sonic, pcaudiolib
 , withPico ? true, svox
+, withIvona ? false # TODO: , libdumbtts
+, defaultModule ? null
+, defaultLang ? "en"
```

Comment:
I would leave this for a NixOS module. Actually, I would even remove the `defaultModule` argument since `selectedDefaultModule` is more of a bugfix, i intend to send a pull request fixing this upstream.

Reply:
Done.